docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -162,9 +163,102 @@ By varying the keyword after the `//` part of the URL in your AMI, you can chang
 
 Using this methodology you could open applications, upload content via services like FTP, archive data, send email, or generate PDF reports.
 
-## Registering a protocol on OSX
+## Registering a protocol on macOS (BigSur and Monterey)
+
+### Create and configure an `.app` bundle
+
+To register a protocol on macOS BigSur and Monterey, you need to create an `.app` bundle that is configured to run your application or script.
+
+**Step 1: AppleScript Script Editor**
+
+First, start by writing the following script in the AppleScript Script Editor:
+
+```
+on open location this_URL
+    do shell script "sgTriggerScript.py '" & this_URL & "'"
+end open location 
+```
+
+> **Note:** To ensure you are running Python from a specific shell, such as `tcsh`, you can change the do shell script for something like the following: do shell script `tcsh -c \"sgTriggerScript.py '" & this_URL & "'\"`. In the Script Editor, save your short script as an _Application Bundle_.
+
+**Step 2: Edit the `info.plist` file**
+
+Find the saved Application Bundle, and Open Contents. 
+
+![Locate App](./images/custom-browser-protocols-right-click-app.png)
+
+Then, open the `info.plist` file and add the following to the plist dict:
+
+![Open info.plist file](./images/custom-browser-protocols-edit-plist.png)
+
+```
+    <key>CFBundleIdentifier</key>
+    <string>com.mycompany.AppleScript.Shotgun</string>
```

Comment:
Should be shotgrid instead of shotgun in this file right?
In the next instructions, we tell the user to use `shotgrid://`

Reply:
It should actually by `foo://` according to the beginning of the document. We'll address that in a further PR.